docs/pages/getting-started.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: Getting started
+description: Getting started with Teleport - identity-aware, multi-protocol access proxy for SSH, HTTPS, Kubernetes and more.
+---
+
+This tutorial will guide you through the steps needed to install and run
+Teleport on Linux machines.
+
+### Prerequisites
+
+- A Linux machine with ports `3023`, `3024`, `3025` and `443` open.
+- A domain name. 
+- Around 20 minutes to complete; half of this may be waiting for DNS propagation.
+
+### Follow along with our video guide
+
+<iframe
+  width="712"
+  height="400"
+  src="https://www.youtube-nocookie.com/embed/yLnaCW8b-8U?rel=0&modestbranding=1"
+  frameBorder="0"
+  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+  allowFullScreen
+/>
+
+## Step 1: Install Teleport on a Linux Host
+
+<Tabs>
+<TabItem label="Amazon Linux 2/RHEL (RPM)">
+```bash
+sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+sudo yum install teleport
+
+# Optional:  Using DNF on newer distributions
+# $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+# $ sudo dnf install teleport
+```
+</TabItem>
+<TabItem label="Debian/Ubuntu (DEB)">
+```bash
+curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+sudo add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+sudo apt-get update
+sudo apt-get install teleport
+```
+</TabItem>
+<TabItem label="Linux">
+```bash
+curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+cd teleport
+sudo ./install
+```
+</TabItem>
+<TabItem label="ARMv7 (32-bit)">
+```bash
+curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-arm-bin.tar.gz
+tar -xzf teleport-v{{ teleport.version }}-linux-arm-bin.tar.gz
+cd teleport
+sudo ./install
+```
+</TabItem>
+<TabItem label="ARMv8 (64-bit)">
+```bash
+curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-arm64-bin.tar.gz
+tar -xzf teleport-v{{ teleport.version }}-linux-arm64-bin.tar.gz
+cd teleport
+sudo ./install
+```
+</TabItem>
+</Tabs>
+
+Take a look at the [Installation Guide](installation.mdx) for more options.
+
+## Step 1b: Configure Teleport
+
+Generate a configuration file for Teleport using `teleport configure`.
+
+Acme turns on automatic TLS certificates from [Letsencrypt](https://letsencrypt.org).
+Set up email to receive updates from Letsencrypt, and use a valid DNS name for a cluster name.
+
+```bash
+$ sudo teleport configure --acme --acme-email=your-email@example.com --cluster-name=tele.example.com -o
+Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
+```
+
+## Step 1c: Configure Domain Name and obtain TLS certificates using Let's Encrypt
```

Comment:
```suggestion
## Step 1c: Update DNS
```

Reply:
I wanted to change all the titles later, because there are other parts of the code referring to those steps, was waiting for linter to land, I will send updates shortly